src/libraries/Microsoft.PowerFx.Repl/Functions/HelpFunction.cs

```diff
@@ -61,21 +61,13 @@ public class HelpProvider
     {
         public const string FormulaRefURL = "https://aka.ms/Power-Fx-Formula-Reference";
 
-        public static IEnumerable<string> FunctionsList(PowerFxREPL repl)
```

Comment:
This is also referenced in the REPL's Program.cs file, it will need to be updated to reference the new function.

Reply:
fixed